index.bs

```diff
@@ -374,6 +374,8 @@ When this method is invoked, the user agent MUST execute the following algorithm
 9. For each authenticator currently available on this platform: asynchronously invoke the <a>authenticatorMakeCredential</a>
     operation on that authenticator with |rpIdHash|, <a>clientDataHash</a>, <a>accountInformation</a>, `normalizedParameters`,
     <a>excludeList</a> and |clientExtensions| as parameters. Add a corresponding entry to |issuedRequests|.
+    - For each credential C in <a>excludeList</a> that has a non-empty |transports| list, use only the specified transports
+       to test for the existence of C.
```

Comment:
Needs more indent to be valid Markdown.

Also, this makes it mandatory for the client to use the hint, maybe we should soften that a bit?